portfolio/src/main/webapp/script.js

```diff
@@ -25,9 +25,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
 //add Hello Nzinga! to the page
-async function getHelloNzingaUsingAsyncAwait() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('HelloNzinga-container').innerText = quote;
+//async function getHelloNzingaUsingAsyncAwait() {
+//  const response = await fetch('/data');
+//  const quote = await response.text();
+//  document.getElementById('HelloNzinga-container').innerText = quote;
+//}
```

Comment:
Please delete, you can find this by going back through your commits later on if needed :)